modules/nf-core/jvarkit/vcffilterjdk/meta.yml

```diff
@@ -0,0 +1,118 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/meta-schema.json
+name: "jvarkit_vcffilterjdk"
+description: Filtering VCF with dynamically-compiled java expressions
+keywords:
+  - vcf
+  - bcf
+  - filter
+  - variant
+  - java
+  - script
+tools:
+  - "jvarkit":
+      description: "Java utilities for Bioinformatics."
+      homepage: "https://github.com/lindenb/jvarkit"
+      documentation: "https://jvarkit.readthedocs.io/"
+      tool_dev_url: "https://github.com/lindenb/jvarkit"
+      doi: "10.1093/bioinformatics/btx734 "
+      licence: ["MIT License"]
+      args_id: "$args2"
+
+  - "bcftools":
+      description: |
+        View, subset and filter VCF or BCF files by position and filtering expression. Convert between VCF and BCF
+      homepage: "http://samtools.github.io/bcftools/bcftools.html"
+      documentation: "http://www.htslib.org/doc/bcftools.html"
+      doi: "10.1093/bioinformatics/btp352"
+      licence: ["MIT"]
+      args_id: ["$args1", "$args3"]
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing VCF information
+        e.g. [ id:'test_reference' ]
+  - vcf:
+      type: file
+      description: Input VCF/BCF file
+      pattern: "*.{vcf,bcf,vcf.gz,bcf.gz}"
+  - tbi:
+      type: file
+      description: Optional VCF/BCF index file
+      pattern: "*.{tbi,csi}"
+  - regions_file:
+      type: file
+      description: Optional. Restrict to regions listed in a file
+      pattern: "*.{bed,bed.gz,txt,tsv}"
+  - meta2:
+      type: map
+      description: |
+        Groovy Map containing fasta information
+        e.g. [ id:'test_reference' ]
+  - fasta:
+      type: file
+      description: Groovy Map containing fasta reference genome information
```

Comment:
You have not changed it since I first commented on this - does the tool expect multiple FASTA files?

Reply:
I'm not sure I understand the problem your comment here. No, there is only one fasta file. Where is the confusion here ?

Reply:
'Groovy Map containing' is what is confusing  me, it's jut the file no?